package.json

```diff
@@ -0,0 +1,70 @@
+{
+	"name": "devdev-kit-js",
```

Comment:
please check npm name.

We need to decide the name to actually publish.

Reply:
Yeah, I think the name is `@devprotocol/dev-kit-js`.

Reply:
I fixed it